draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1303,6 +1303,45 @@ When possible, developers should use a session management mechanism such as
 that described in {{top-level-navigations}} to mitigate the risk of CSRF more
 completely.
 
+#### "Lax-Allowing-Unsafe" enforcement {#lax-allowing-unsafe}
+
+As discussed in {{unsafe-top-level-requests}}, compatibility concerns may
+necessitate the use of a "Lax-allowing-unsafe" enforcement mode that allows
+cookies to be sent with a cross-site HTTP request if and only if it is a
+top-level request, regardless of request method. That is, the
+"Lax-allowing-unsafe" enforcement mode waives the requirement for the HTTP
+request's method to be "safe" in the `SameSite` enforcement step of the
+retrieval algorithm in {{retrieval-algorithm}}. (All cookies, regardless of
+`SameSite` enforcement mode, may be set for top-level navigations, regardless of
+HTTP request method, as specified in {{storage-model}}.)
+
+"Lax-allowing-unsafe" is not a distinct value of the `SameSite` attribute.
+Rather, user agents MAY apply "Lax-allowing-unsafe" enforcement only to cookies
+that did not explicitly specify a `SameSite` attribute (i.e., those whose
+same-site-flag was set to "Default" by default). If a user agent applies
+"Lax-allowing-unsafe" enforcement, the user agent SHOULD restrict
+"Lax-allowing-unsafe" enforcement to cookies whose age is less than a duration
+of the user agent’s choosing (2 minutes seems reasonable).
```

Comment:
I think the wording here is a bit open ended. How about:

"To limit the scope of this compatibility mode, user agents which apply "Lax-allowing-unsafe" SHOULD restrict the enforcement to cookies which were created recently. Deployment experience has shown that a cookie age of 2 minutes or less to be a reasonable limit."

Reply:
Done.